db/migrations/20190807230019-add-indexes.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addIndex('Subscriptions', {
+      fields: ['gitHubInstallationId', 'jiraHost'],
+      name: 'index-on-installation-and-jirahost'
+    })
+
+    await queryInterface.addIndex('Subscriptions', {
+      fields: ['gitHubInstallationId'],
```

Comment:
If I understand correctly, the composite index from line 5 will support querying against _only_ `gitHubInstallationId`.

```sql
SELECT * FROM "Subscriptions" WHERE "gitHubInstallationId" = 123; -- covered by index
SELECT * FROM "Subscriptions" WHERE "gitHubInstallationId" = 123 AND jiraHost = "github.jira.com"; -- covered by index
SELECT * FROM "Subscriptions" WHERE jiraHost = "github.jira.com"; -- NOT covered by index
```

Do we ever query using only the `jiraHost`? If so, what do you think about adding that as a separate index?

Reply:
> Do we ever query using only the jiraHost? If so, what do you think about adding that as a separate index?

Yeh working on that now. We also need one for the jira client key. Multi-column indexes always make my head hurt, so thanks!